### PR TITLE
(New) Many WMP9 Video Playback fixes for Yuzusoft VN's

### PR DESCRIPTION
**Pull request by CreeperofDoom**
This will fix 100% WMP9 Video Playback for many Yuzusoft VN's, Including Riddle Joker. I am very certain this works, excluding even a single command will either cause a black screen, laggy or no audio or it just wont play. (I tested this very well)

### CODE REVIEW

**Comment by CreeperofDoom** [comment]
Failed? Nothing is wrong with it, what am i supposed to change here?

**Comment by R1kaB3rN** [comment]
Thanks for the pull request. I'll test them in a bit since I own them, and I think it would probably be a good idea to create fixes in GOG or umu because they're available in other storefronts -- GOG and NekoNyan

File: gamefixes-steam/1144400.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+""" Game fix for Senren Banka
+"""
+#pylint: disable=C0103
+
+from protonfixes import util
+
+def main():
+    """
+    """
```
Comment (R1kaB3rN) [inline gamefixes-steam/1144400.py:9]: It's the empty docstring which is unnecessary. See the linter errors for more details

**Comment by R1kaB3rN** [comment]
To clarify, this fixes the video playback specifically for the opening and ending videos, correct?

**Comment by CreeperofDoom** [comment]
> To clarify, this fixes the video playback specifically for the opening and ending videos, correct?

Yes, ive tested. In the normal playthrough they play with full video and sound. Replaying them through the scene select also has full video and sound

File: gamefixes-steam/1144400.py (hunk 1 above)
Comment (CreeperofDoom) [reply gamefixes-steam/1144400.py:9]: and does this mean i have to submit another pull request with those gone in the script

**Comment by CreeperofDoom** [comment]
i will now submit yet another Pull Request with the spaces gone so it can pass the checks.

**Pull request closed without merging**